app/src/main/res/values/strings.xml

```diff
@@ -45,7 +45,7 @@
     <string name="invalid_url">Invalid URL</string>
     <string name="subject">Topic</string>
     <string name="reply_to_sender">Reply to sender</string>
-    <string name="google_app_login_failed">Google apps login failed for unknown reasons, please contact your distributor.</string>
+    <string name="google_app_login_failed">Google Apps login failed for unknown reasons, please contact your distributor.</string>
```

Comment:
We follow the Capitalization scheme's as discussed here, so please follow this https://github.com/zulip/zulip-android/issues/405